python/test/test_polars.py

```diff
@@ -188,14 +183,14 @@ def test_cast():
 
 def test_stable_expr():
     pl = pytest.importorskip("polars")
-    domain = dp.expr_domain(example_lf()[0])
+    domain = dp.wild_expr_domain(example_series()[0])
```

Comment:
It looks like we never referenced `expr_domain` in the docs, except for the generated API docs. If users would never need to use `expr_domain` or `wild_expr_domain` directly, then maybe they should be private? (Or else docs that explain when they would use it.)

(Started wondering about whether this is the best name in a public interface, but if it's something that only gets used internally, it's a fine name.)

Reply:
I will add a docs page for it